config.txt

```diff
@@ -0,0 +1,23 @@
+# database
```

Comment:
we already have a config file that does exactly this! please look at the `credentials.json` file that I shared in discord (make sure to not commit it since there are passwords in it)